lang/nl/strings.php

```diff
@@ -58,21 +76,25 @@
     'action' => 'Actie',
     'data' => 'Gegevens',
     'queued' => 'In wachtrij',
-    'last_run' => 'Laatst uitgevoerd',
-    'next_run' => 'Volgende uitvoering',
-    'not_run_yet' => 'Nog niet uitgevoerd',
+    'last_run' => 'Laatst Uitgevoerd',
```

Comment:
Please be consistent in your capitalisation,

Reply:
It should be capitalized, tho. Since the original one was also